app/Http/Controllers/Medication/MedicationController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Controllers\Medication;
+
+use App\Http\Controllers\Api\ApiResponser;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Medication\CreateMedicationRequest;
+use App\Http\Resources\Medication\MedicationResource;
+use App\Services\Medication\MedicationCreator;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class MedicationController extends Controller
+{
+    use ApiResponser;
+    /**
+     * Show the form for creating a new resource.
+     * @param CreateDoctorRequest $request
```

Comment:
Should be CreateMedicationRequest for this you should us 
php artisan make:controller ControllerName --resource